terraso_backend/apps/project_management/models/sites.py

```diff
@@ -65,9 +57,6 @@ class Meta(BaseModel.Meta):
 
     PRIVACY_STATUS = ((PRIVATE, _("Private")), (PUBLIC, _("Public")))
 
-    # changing settings fields must be restricted by the corresponding permission
-    SETTINGS_FIELDS = set(["name", "privacy", "projectId"])
```

Comment:
This field is now unused -- see the comment I left in in `graphql/schema/sites.py`.